cache-config/testing/ort-tests/tcdata/roles.go

```diff
@@ -30,15 +30,33 @@ const (
 )
 
 func (r *TCData) CreateTestRoles(t *testing.T) {
-	expectedAlerts := []tc.Alerts{tc.Alerts{Alerts: []tc.Alert{tc.Alert{Text: "role was created.", Level: "success"}}}, tc.Alerts{Alerts: []tc.Alert{tc.Alert{Text: "can not add non-existent capabilities: [invalid-capability]", Level: "error"}}}, tc.Alerts{Alerts: []tc.Alert{tc.Alert{Text: "role was created.", Level: "success"}}}}
+	expectedAlerts := []tc.Alerts{
+		{
+			Alerts: []tc.Alert{{
+				Text:  "role was created.",
+				Level: "success",
+			}},
+		},
+		{
+			Alerts: []tc.Alert{{
+				Text:  "can not add non-existent capabilities: [invalid-capability]",
+				Level: "error",
+			}},
+		},
+		{
+			Alerts: []tc.Alert{{
+				Text:  "role was created.",
+				Level: "success"}},
+		},
+	}
 	for i, role := range r.TestData.Roles {
 		var alerts tc.Alerts
 		alerts, _, status, err := TOSession.CreateRole(role)
 		t.Log("Status Code: ", status)
 		t.Log("Response: ", alerts)
 		if err != nil {
+			// TODO: Why is this not a failure condition?
```

Comment:
I believe this failed at one point, but the t3c tests didn't actually need it. 
If it's passing now, I think it should just be made a real error.
If it's still not passing, the TODO should probably be to figure out and fix.

Reply:
On further investigation I know why, actually. It's because the tests are testing creation of invalid Roles, which are expected to fail, but in those cases only checks against the expected Alerts. t3c doesn't - and shouldn't - rely on those Roles because those Roles were never actually meant to be successfully created. Also, though, even testing that behavior at all is a waste of the t3c tests' time; it's already covered elsewhere and doesn't fit the purpose of the tests.

There's a lot of stuff like that left over from the copy/paste from the API tests. I may make a follow-up PR to get rid of more of it, at some point.